TheForceEngine/TFE_Audio/midiPlayer.cpp

```diff
@@ -505,10 +521,27 @@ namespace TFE_MidiPlayer
 
 				// Check for hanging notes.
 				detectHangingNotes();
+
+				// Sleep until the next playback window to
+				// avoid a busy waiting scenario.
+				auto timeout = (s_midiCallback.timeStep - s_midiCallback.accumulator) * 1000;
+				if (!std::isfinite(timeout))
```

Comment:
This doesn't work with VisualStudio, std::isfinite() is unrecognized by the msvc++ compiler.

Reply:
Hey, which VS is this?

Reply:
I tried with VS2022, SDK10, v143 (the latest available), msvc++ compiler.

Reply:
Interesting... anyhow I refactored the code so it's not used any more.